### PR TITLE
Minor clean-up translation vignette setup

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
Hopefully, this slightly improves the setup chunk code, irrespective of whether it solves #527 or not.

### CODE REVIEW

File: vignettes/tidyverse_translation.Rmd [hunk 1]
```diff
@@ -21,16 +21,22 @@ knitr::opts_chunk$set(
 
 pkgs <- c(
   "dplyr",
-  "datawizard",
```
Comment (IndrajeetPatil) [inline vignettes/tidyverse_translation.Rmd:24]: No need to check if `{datawizard}` is available.

File: vignettes/tidyverse_translation.Rmd [hunk 2]
```diff
@@ -39,7 +47,7 @@ This vignette can be referred to by citing the following:
 
 Patil et al., (2022). datawizard: An R Package for Easy Data Preparation and Statistical Transformations. *Journal of Open Source Software*, *7*(78), 4684, https://doi.org/10.21105/joss.04684
 
-```{css, echo=FALSE, eval = evaluate_chunk}
+```{css, echo=FALSE, eval = TRUE}
```
Comment (IndrajeetPatil) [inline vignettes/tidyverse_translation.Rmd:50]: No need to eval CSS code conditionally. Doesn't hurt us if it is evaluated.

File: vignettes/tidyverse_translation.Rmd [hunk 3]
```diff
@@ -84,10 +92,6 @@ This vignette is largely inspired from `{dplyr}`'s [Getting started vignette](ht
 </div>
 
 ```{r, eval = evaluate_chunk}
-library(dplyr)
-library(tidyr)
-library(datawizard)
```
Comment (IndrajeetPatil) [inline vignettes/tidyverse_translation.Rmd:89]: We always load all needed libraries in the setup chunk. Happy to revert this if you think this reduces readability of the vignette.

File: vignettes/tidyverse_translation.Rmd [hunk 4]
```diff
@@ -21,16 +21,25 @@ knitr::opts_chunk$set(
...
+can_vignette_be_evaluated <- all_deps_available && getRversion() >= "4.1.0"
 
-if (!all(vapply(pkgs, requireNamespace, quietly = TRUE, FUN.VALUE = logical(1L))) || getRversion() < "4.1.0") {
+# Since we explicitly put `eval = TRUE` for some chunks, we can't rely on
+# `knitr::opts_chunk$set(eval = FALSE)` at the beginning of the script.
+# Therefore, we introduce a logical that is `FALSE` only if all suggested
+# dependencies are not installed (cf easystats/easystats#317)
+if (can_vignette_be_evaluated) {
+  evaluate_chunk <- TRUE
+} else {
```
Comment (etiennebacher) [inline vignettes/tidyverse_translation.Rmd:42]: I don't understand this. Why not put `evaluate_chunk <- all_deps_available && getRversion() >= "4.1.0"` and remove this `if` condition?

**Review by etiennebacher — CHANGES_REQUESTED** [review]

File: vignettes/tidyverse_translation.Rmd (hunk 4 above)
Comment (IndrajeetPatil) [reply vignettes/tidyverse_translation.Rmd]: Good point. No need for an extra level of indirection.

**Review by etiennebacher — APPROVED** [review]
Thanks @IndrajeetPatil

**Pull request merged**